lib/hanami/action/response.rb

```diff
@@ -81,6 +81,23 @@ def body=(str)
         end
       end
 
+      # Accept a symbolic status code
+      #
+      # @param code [Integer, Symbol] the status code
+      #
+      # @example
+      #   response.status = :unprocessable_entity
+      #
```

Comment:
@alassek How a developer gets the list of possible status codes?
Please open a PR in https://github.com/hanami/guides to document those.

Add here the following code:

```ruby
# @see link to guides URL
```